docs/src/pages/components/data-grid/columns/StylingHeaderGrid.js

```diff
@@ -7,11 +7,19 @@ const columns = [
     field: 'first',
     headerClassName: 'super-app-theme--header',
     headerAlign: 'center',
+    hideSortIcons: 'unsorted',
   },
   {
     field: 'last',
     headerClassName: 'super-app-theme--header',
     headerAlign: 'center',
+    hideSortIcons: true,
```

Comment:
I like the idea but I was thinking of a different API. What if we have a prop like `sortType` and based on its value we choose the icon. So for example possible values can be `dec | asc | unsorted | none`?